src/components/Square.jsx

```diff
@@ -1,16 +1,14 @@
-import React, { useState } from 'react';
+import React from 'react';
 
-const Square = ({ value }) => {
-
-    let [innerValue, setInnerValue] = useState(null);
+const Square = ({ value, onClick }) => {
 
     return (
         <>
             <button
```

Comment:
No need for a fragment when you return just one element (`button` in this case).

```suggestion
    return (
          <button
```